src/linked-list.js

```diff
@@ -4,6 +4,40 @@ class LinkedList {
     this.tail = null;
     // Do not modify anything inside of the constructor
   }
+
+  addToTail(element) {
+    const node = {
+      value: element,
+      next: null
+    };
+
+    if (this.head === null) {
+      this.head = node; // correct
+      this.tail = node;
```

Comment:
Since this is already being written after the if/else, you could remove this line.